docs/docs/08.1-more-about-refs.md

```diff
@@ -1,75 +1,76 @@
 ---
 id: more-about-refs
-title: More About Refs
+title: Refs to Components
 permalink: more-about-refs.html
 prev: working-with-the-browser.html
 next: tooling-integration.html
 ---
-After returning the structure of your UI from the render method, you may find yourself wanting to "reach out" and invoke methods on component instances returned from `render()`. Often, doing something like this isn't necessary for making data flow through your application, because the Reactive data flow always ensures that the most recent `props` are sent to each child that is output from `render()`. However, there are a few cases where it still might be necessary or beneficial.
+After building your component, you may find yourself wanting to "reach out" and invoke methods on component instances returned from `render()`.  In most cases, this is ideally unnecessary (and should be avoided) because the reactive data flow always ensures that the most recent props are sent to each child that is output from render().  However, there are a few cases where it still might be necessary or beneficial, so React provides an escape hatch known as `refs`.  These `refs` (references) are especially useful when you need to: find the DOM markup rendered by a component (for instance, to position it absolutely), use React components in a larger non-React application, or transition your existing codebase to React.
```

Comment:
> In most cases, this is ideally unnecessary (and should be avoided) because the reactive data flow always ensures that the most recent props are sent to each child that is output from render().

There are a lot of words in there.

> In most cases this is unnecessary because …

Also, maybe put render() in backticks since you did in the previous sentence.